bark/models/behavior/action_store/behavior_action_store.hpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019 fortiss GmbH, Julian Bernhard, Klemens Esterle, Patrick
+// Hart, Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#ifndef MODULES_MODELS_BEHAVIOR_ACTION_STORE_BEHAVIOR_ACTION_STORE_HPP_
+#define MODULES_MODELS_BEHAVIOR_ACTION_STORE_BEHAVIOR_ACTION_STORE_HPP_
+
+#include <boost/functional/hash.hpp>
+#include <unordered_map>
+
+#include "bark/models/behavior/behavior_model.hpp"
+
+namespace bark {
+namespace world {
+class ObservedWorld;
+}  
+namespace models {
+namespace behavior {
+
+
+using dynamic::Trajectory;
+using dynamic::Input;
+
+
+class BehaviorActionStore : public BehaviorModel {
+ public:
+  BehaviorActionStore(const commons::ParamsPtr& params)
+      : BehaviorModel(params),
```

Comment:
the clang formatter does not seem to work correctly. maybe talk to klemens about this?